README.md

```diff
@@ -38,9 +38,36 @@ pip3 install -e .
 
 Note that you need to install protobuf before building from source.
 
+
+## Command-line API
+Now you can use command-line api in terminal instead of  python script.
+
+```
+python -m onnxoptimizer -i <input-modle-file> -o <output-model-file>
```

Comment:
The following command (without -i and -o) is better:

```bash
python -m onnxoptimizer <input-model-file> <output-model-file>
```

And please consider following https://github.com/daquexian/onnx-simplifier/blob/master/setup.py#L303-L307 to make this possible:
```bash
onnxoptimizer <input-model-file> <output-model-file>
```